src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -166,6 +166,15 @@ pip3 install -r requirements.txt
 $ cwltool --version
 /usr/local/bin/cwltool ${this.cwltoolVersion}
 \`\`\`
+
+#### Part 5 - Install Nextflow (Optional)
+The Dockstore CLI does not run Nextflow workflows. Users must run them directly by using the Nextflow command line tool.
+1. You can install the Nextflow CLI with the following command:
```

Comment:
Just my opinion of course, but I rather link out to the actual nextflow documentation because I wouldn't want to maintain this (in case they somehow change the installation instructions)

https://github.com/nextflow-io/nextflow#download-the-package

Approved assuming I don't need to maintain this.